x-pack/plugins/security_solution/public/detections/components/alerts_kpis/common/types.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export interface AlertsStackByOption {
+  text: AlertsStackByField;
+  value: AlertsStackByField;
+}
+
+export type AlertsStackByField =
```

Comment:
Maybe in a follow-up PR can you add a comment saying why these fields specifically? Thanks!

Reply:
I asked myself the same question. "Why these fields?".
The only answer I got is that these values are hardcoded since the first implementation. The original PR that introduced this feature lists them https://github.com/elastic/kibana/pull/53742.

Maybe someone else could help us here. 😵‍💫